plugins/uberdev/lib/goal-abort.sh

```
@@ -0,0 +1,211 @@
+fi
+
+if [ "$UBERDEV_GOAL_ABORT_DRY_RUN" = "1" ]; then
+  printf 'goal-abort: DRY RUN goal=%s would_release=%d skipped=%d (no GitHub mutation, run-state kept)\n' \
+    "$GOAL_ID" "$RELEASED" "$SKIPPED"
+  exit 0
+fi
+
+if [ "$FAILED" -gt 0 ]; then
+  printf 'goal-abort: goal=%s released=%d FAILED=%d skipped=%d — run-state KEPT so this can be retried\n' \
+    "$GOAL_ID" "$RELEASED" "$FAILED" "$SKIPPED"
+  printf 'goal-abort: still claimed:%s — release by hand with `gh issue edit N --remove-label uberdev:active`\n' \
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)